sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/java/JavaGenerator.java

```diff
@@ -60,6 +61,7 @@ public class JavaGenerator implements CodeGenerator
     private static final String PACKAGE_INFO = "package-info";
     private static final String BASE_INDENT = "";
     private static final String INDENT = "    ";
+    private static final Set<String> NO_PACKAGES = Collections.unmodifiableSet(new HashSet<>());
```

Comment:
Why not EMPTY_SET. https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptySet()

Reply:
No idea, really. I use it all the time. Fixed.